src/test/java/io/nats/client/impl/MessageQueueTests.java

```diff
@@ -26,14 +26,15 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 public class MessageQueueTests {
-    byte[] PING = "PING".getBytes();
-    byte[] ONE = "one".getBytes();
-    byte[] TWO = "two".getBytes();
-    byte[] THREE = "three".getBytes();
+    static final Duration REQUEST_CLEANUP_INTERVAL = Duration.ofSeconds(5);
+    static final byte[] PING = "PING".getBytes();
+    static final byte[] ONE = "one".getBytes();
+    static final byte[] TWO = "two".getBytes();
+    static final byte[] THREE = "three".getBytes();

```

Comment:
1. Just cleanup.
2. All other changes in this test file are b/c of the MessageQueue constructor change